src/GitHub.VisualStudio/Settings/PackageSettings.cs

```diff
@@ -10,13 +11,61 @@
 namespace GitHub.VisualStudio.Settings
 {
     [Export(typeof(IPackageSettings))]
-    [PartCreationPolicy(CreationPolicy.Shared)]
+    [PartCreationPolicy(CreationPolicy.NonShared)]
+    public class PackgeSettingsDispatcher : IPackageSettings
```

Comment:
I think we can use the technique from #1518 for this too.

Reply:
👍